Bottom: 57791848213d1589f05cf4991414369d8c03a286
Top:    8ccd592d8879f50f8b14630694d27daaf8c3dcc4
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-07 10:33:33 +0800

Refresh of autofs-5.0.7-improve-timeout-option-description.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index c3f12e5..9639703 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -73,6 +73,7 @@
 - teach dumpmaps to output simple key value pairs.
 - fix syncronize handle_mounts() shutdown.
 - fix fix wildcard multi map regression.
+- improve timeout option description.
 
 25/07/2012 autofs-5.0.7
 =======================
